Mathlib/Data/Nat/Bitwise.lean

```diff
@@ -1,17 +1,17 @@
 /-
 Copyright (c) 2020 Markus Himmel. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Markus Himmel
+Authors: Markus Himmel, Harun Khan, Abdalrhman M Mohamed
 -/
 import Lean.Elab.Tactic
 import Mathlib.Data.List.Basic
+import Mathlib.Data.Bool.Basic
+import Mathlib.Data.Nat.ModEq
 import Mathlib.Data.Nat.Bits
 import Mathlib.Data.Nat.Size
 import Mathlib.Data.Nat.Order.Lemmas
 import Mathlib.Tactic.Linarith
 
-#align_import data.nat.bitwise from "leanprover-community/mathlib"@"6afc9b06856ad973f6a2619e3e8a0a8d537a58f2"
```

Comment:
Accidental deletion?

Reply:
Is this the only remaining issue? Or is there sth else to be done in this PR?

Reply:
I would prefer to see a separate PR that addresses https://github.com/leanprover-community/mathlib4/pull/5920#discussion_r1310496092 first; though maybe another reviewer can give a second opinion.

Reply:
I've marked this unresolved again because this line is still deleted and shouldn't be.

Reply:
This has now been reverted.